@ecomplus/storefront-components/src/js/RecommendedItems.js

```diff
@@ -62,16 +61,7 @@ export default {
         return ecomCart
       }
     },
-    ecomPassport: {
-      type: Object,
-      default () {
-        return ecomPassport
-      }
-    },
-    isFavoritesTab: {
-      type: Boolean,
-      default: false
-    }
+    favoriteItems: []
```

Comment:
```suggestion
    items: {
      type: Array,
      default () {
        return []
      }
    }
```

não necessariamente são favoritos, a da sugestão anterior é justamente pro componente funcionar também em outros casos passando os items em prop.
além disso a prop estava declarada de um jeito estranho aqui, não sei se funcionaria exatamente da mesma forma, mas acho que não 😐

Reply:
aí o `items` no data você muda pra `localItems` por exemplo, é mais importante a nomenclatura na API exposta do componente (props, events, slots)